build.rs

```diff
@@ -545,9 +573,14 @@ fn cc(
     target: &Target,
     warnings_are_errors: bool,
     out_dir: &Path,
+    definitions: &[(&str, String)],
 ) -> Command {
     let mut c = cc::Build::new();
     let _ = c.include("include");
+    if std::env::var("CARGO_FEATURE_BORINGSSL_NO_PREFIX").is_err() {
+        let _ = c.include(format!("{}/{}", PREGENERATED, SYMBOL_PREFIX_INCLUDE));
+    }
```

Comment:
I would rather have this logic within `base.h`, which is included by every C file.

Reply:
this just adds the includes directory to the search path. it doesn't actually include any of the headers. I think without this conditional, the build fails when the directory doesn't exist. I can guard it with a check for the directory existing instead of the environment variable, if you prefer.